src/YesSql.Core/Sql/ISqlDialect.cs

```diff
@@ -72,6 +74,9 @@ public abstract class BaseDialect : ISqlDialect
 
         public virtual string PrimaryKeyString => "primary key";
 
+        public virtual char OpenQuote => '[';
```

Comment:
These should juts be constants. We can already override the methods that use them, and in terms of micro-performance it will be more optimal than calling a virtual member twice more.